cristin-project/src/main/java/no/unit/nva/cristin/projects/query/QueryCristinProjectHandler.java

```diff
@@ -87,82 +50,28 @@ protected QueryCristinProjectHandler(QueryCristinProjectApiClient cristinApiClie
     protected SearchResponse<NvaProject> processInput(Void input, RequestInfo requestInfo, Context context)
             throws ApiGatewayException {
 
-        validateQueryParameterKeys(requestInfo);
-
-        var requestQueryParameters = extractQueryParameters(requestInfo);
-        if (requestInfo.getQueryParameters().containsKey(ORGANIZATION)) {
-            requestQueryParameters.put(ORGANIZATION, getValidOrganizationUri(requestInfo));
-        }
-        getValidProjectStatus(requestInfo).ifPresent(status -> requestQueryParameters
-                .put(STATUS, status.name()));
-        return getTransformedCristinProjectsUsingWrapperObject(requestQueryParameters);
+        final var cristinQuery =
+            CristinQuery.builder()
+                .fromRequestInfo(requestInfo)
+                .withRequiredParameters(REQUIRED_QUERY_PARAMETER)
```

Comment:
Are some params now required or do we use a default value just like before?

Reply:
Only required parameters can have their default values set (a parameter that is not required should not be added if absent)